_posts/en/newsletters/2020-12-02-newsletter.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Bitcoin Optech Newsletter #126'
+permalink: /en/newsletters/2020/12/02/
+name: 2020-12-02-newsletter
+slug: 2020-12-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal for using fidelity bonds on
+LN to prevent denial of service attacks, summarizes a PR to address a
+fee siphoning attack that could affect LN channels using anchor outputs,
+and links to a proposed specification for miniscript.  Also included are
+our regular sections with releases, release candidates, and recent code
+changes in popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Fidelity bonds for LN routing:** Gleb Naumenko and Antoine Riard
+  [posted][gnar post] a to the Lightning-Dev mailing list to use *stake
+  certificates* (another name for fidelity bonds) to prevent a type of
+  channel jamming attack [first described in 2015][russell loop].
+  These are attacks where a malicious user sends a payment to themselves
+  or a confederate through a series of channels and then delays either
+  accepting or rejecting the payment.  Until the payment eventually
+  times out, each channel used to route the payment is unable to use
+  those funds to route other user's payments.  Since a route may cross
+  more than a dozen channels, that means every bitcoin controlled by the
+  attacker can prevent more than a dozen bitcoins belonging to honest
+  nodes from being used for honest routing. <!-- "more than a dozen" is conservative, I
+  think 25 is the actual max:
+  http://gnusha.org/lightning-dev/2020-11-30.log 04:42 -->
+
+    Previously proposed solutions for this problem (and related
+    problems) mostly involved upfront fees, see Newsletters
+    [#72][news72 upfront], [#86][news86 upfront], [#119][news119
+    upfront], [#120][news120 upfront], and [#122][news122 upfront].
+    This week, Naumenko and Riard proposed that each payment include
+    proof that its spender controlled some amount of bitcoin.  Each
+    routing node could then publicly announce its policy on how much
+    value it would route given a proof of a certain stake value.  For
```

Comment:
nit: I realize you are referring to _a_ proof but I think it reads better as proof as in evidence of
```suggestion
    value it would route given proof of a certain stake value.  For
```